parachain/pallets/verifier-lightclient/Cargo.toml

```diff
@@ -12,12 +12,21 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 hex = { package = "rustc-hex", version = "2.1.0", default-features = false }
 
+[dev-dependencies]
+serde_json = "1.0"
+artemis-testutils = { path = "../../primitives/testutils" }
+
 [dependencies.codec]
 default-features = false
 features = ["derive"]
 package = "parity-scale-codec"
 version = "1.3.4"
 
+[dependencies.ethash]
+default-features = false
+git = "https://github.com/nearprotocol/rust-ethash"
+branch = "upgrade-eth-types"
```

Comment:
Should I create an issue to pin this version?

Reply:
yeah, I think go ahead and create an issue for that. I think it may be safer to fork their repo though, since in this case we're depending on a development branch of someone else's repo. That branch could be deleted or updated under our feet at any time.

Reply:
Let me actually check if tests pass with the latest published version. It uses an old version of `ethereum-types` and `primitive-types`.

Reply:
Nope. I'll make an issue

Reply:
https://github.com/Snowfork/polkadot-ethereum/issues/148